packages/remix-eslint-config/package.json

```diff
@@ -44,13 +44,17 @@
     "@types/eslint": "^8.4.1"
   },
   "peerDependencies": {
-    "eslint": ">=8.1.0",
-    "react": ">=17.0.0",
-    "react-dom": ">=17.0.0"
+    "eslint": "^8.0.0",
+    "react": "^17.0.0 || ^18.0.0",
+    "react-dom": "^17.0.0 || ^18.0.0",
+    "typescript": "^4.0.0"
```

Comment:
It was in `peerDependenciesMeta`, but not in `peerDependencies`

Reply:
@MichaelDeBoey This is intentional. Users who don't use TypeScript but want to use our rules shouldn't get bugged about missing peer deps that aren't applicable to them.

Reply:
> Users who don't use TypeScript but want to use our rules shouldn't get bugged about missing peer deps that aren't applicable to them.

That's why it's marked as `optional` in `peerDependenciesMeta`, no? 🤔
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta

Reply:
@MichaelDeBoey Gah, sorry about that. Misread the diff, thanks for clarifying 👍